app/data/functions.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+from app.data.knight_class import Knight
+
+
+def knight_the_knight(data: dict) -> Knight:
+    knight = Knight(
+        data["name"],
+        data["power"],
+        data["hp"]
+    )
+
+    knight.grab_weapon(data["weapon"])
+    knight.suit_up(data["armour"])
+    knight.drink_potion(data["potion"])
+
+    return knight

```

Comment:
It is generally not a good idea to name   a module as functions in Python because it can lead to confusion.

Reply:
renamed
thanks for the comment